client/src/ui/deckbuilder/DeckBuilderColumn.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div
+      class="column"
+      @dragover="dragOver"
+      @dragleave="dragEnd"
+      @dragend="dragEnd"
+      @drop="drop"
+  >
+    <div
+        v-for="(card, index) in column"
```

Comment:
v-for elements should always also have an accompanying `:key=...` attribute (to assist in the diffing algorithm). `card.id` is probably the best fit.

Reply:
`card.id` doesn't exist here so I  had to improvise.